frontend/device/sysfs_light.lua

```diff
@@ -56,29 +57,38 @@ function SysfsLight:setNaturalBrightness(brightness, warmth)
         warmth = self.current_warmth
     end
 
-    local red = 0
-    local green = 0
-    local white = 0
-    if brightness > 0 then
-        -- On Nickel, the values for white/red/green are roughly linearly dependent
-        -- on the 4th root of brightness and warmth.
-        white = math.min(self.white_gain * math.pow(brightness, self.exponent) *
+    -- Newer devices use a mixer instead of writting values per color.
+    if self.frontlight_mixer then
+        -- And the warmth is now on a scale of 0 to 10 ;).
+        warmth = math.floor(warmth/10)
+        self:_write_value(self.frontlight_white, brightness)
```

Comment:
can we use `self:_set_light_value(self.frontlight_white, brightness)` and leave frontlight_white as is?

Reply:
I mean, writting bl_power is not neccesary but shouldn't hurt.

Reply:
`-1676681792`, to be exact ;p.

```
┌─(ROOT@(none):pts/0)───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────(/)─┐
└─(0.22:58%:22:28:96%:#)── cat /sys/class/backlight/mxc_msp430.0/bl_power                                                                                                                                                                                                                                                                               ──(Sat, Dec 22)─┘
-1676681792
┌─(ROOT@(none):pts/0)───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────(/)─┐
└─(0.20:58%:22:28:96%:#)── cat /sys/class/backlight/tlc5947_bl/bl_power                                                                                                                                                                                                                                                                                 ──(Sat, Dec 22)─┘
0
```

Reply:
Given that, I refrained from trying to write to it ;p.